packages/dialtone-vue2/components/filter_pill/filter_pill.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <dt-popover
```

Comment:
Does this restrict it directly just Popover? Might be a Dropdown, which still is a Popover. I can't think of another thing it would trigger though.

Reply:
Yes, it's restricted directly to just the popover slots, when refining the ticket we didn't found other usages that were not using popover so we decided to make it this way.

Reply:
Would it work with DtDropdown? Current Filter Pills may not invoke Dropdown, but it definitely should be an option.

Reply:
That could be a future improvement if needed, I would need to reestructure the component right now to add that support